src/main/java/com/blamejared/jeitweaker/zen/component/HackyJeiIngredientToMakeZenCodeHappy.java

```diff
@@ -0,0 +1,19 @@
+package com.blamejared.jeitweaker.zen.component;
+
+import com.blamejared.crafttweaker.api.annotations.ZenRegister;
+import com.blamejared.crafttweaker.api.brackets.CommandStringDisplayable;
+import com.blamejared.crafttweaker_annotations.annotations.Document;
+import org.openzen.zencode.java.ZenCodeType;
+
+@Document("mods/JEITweaker/Component/JeiIngredient")
+@ZenCodeType.Name("mods.jei.component.JeiIngredient")
+@ZenRegister
+// TODO("Replace all usages of this with JeiIngredient once generic inference is better")
+public interface HackyJeiIngredientToMakeZenCodeHappy extends CommandStringDisplayable {
```

Comment:
Not super keen on the name, it is descriptive sure, but it is going to be printed to logs potentially